topology/pre-process-object.c

```diff
@@ -934,6 +935,23 @@ static int tplg_build_generic_object(struct tplg_pre_processor *tplg_pp, snd_con
 	return ret;
 }
 
+static int tplg_update_dai_auto_attr(struct tplg_pre_processor *tplg_pp,
+				     snd_config_t *dai_cfg, snd_config_t *parent)
+{
+	const char *id;
+	int ret;
+
+	ret = snd_config_get_id(dai_cfg, &id);
+	if (ret < 0)
+		return ret;
+
+	tplg_pp_debug("entering dai auto attr id %s", id);
```

Comment:
can we clean up the debug prints a bit to keep only the most critical ones. I understand these are useful for you while you're inplementing the feature but they may pollute the output when building with debug enabled for the end user

Reply:
yes, will clean